ecommerce-demo/workspaces/childDemo_electronics/workspaceDataFilterSettings.json

```diff
@@ -0,0 +1,45 @@
+{
+  "workspaceDataFilterSettings": [
+    {
+      "data": {
+        "type": "workspaceDataFilterSetting",
+        "id": "ecommerce-merchant-electronics",
```

Comment:
Should we add `demo_wdfSettings` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_wdfSettings_ecommerce_merchant_electronics",
```